crates/builder/src/generate/output.rs

```diff
@@ -141,11 +136,11 @@ impl Output {
         let dir = self.full_created_dir(dir)?;
 
         let hash = self.checksum.then(|| {
-            let mut checksummer = SeaHasher::new();
-            variants
-                .iter()
-                .for_each(|(_, content)| checksummer.write(content));
-            URL_SAFE.encode(checksummer.finish().to_be_bytes())
+            let contents = variants.iter().fold(Vec::new(), |mut acc, (_, contents)| {
+                acc.extend_from_slice(contents);
+                acc
+            });
+            URL_SAFE.encode(seahash::hash(&contents).to_be_bytes())
```

Comment:
I would be very worried if this change is necessary. I.e. if seahash needs all the content in one vec. The foundation of how it works is by adding bytes to the hash (impl Hash does exactly that).

Please revert it to make sure that there is not something really weird here.

Reply:
@anotherempty this is still pending, right?

Reply:
it's done